tests/baselines/reference/additionOperatorWithNullValueAndInvalidOperator.errors.txt

```diff
@@ -23,37 +23,37 @@ tests/cases/conformance/expressions/binaryOperators/additionOperator/additionOpe
     
     // null + boolean/Object
     var r1 = null + a;
-             ~~~~~~~~
-!!! error TS2365: Operator '+' cannot be applied to types 'boolean' and 'boolean'.
+             ~~~~
+!!! error TS2531: Object is possibly 'null'.

```

Comment:
Object is *definitely* `null`, and typically we don't think of objects as being operands to arithmetic operations (whereas we do when it comes to primitives). Would be nice if we could be more precise on the error message.